.github/workflows/tekton_bundle_push.yaml

```diff
@@ -0,0 +1,117 @@
+---
+name: Tekton Bundle Push
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['main']
+  workflow_dispatch:
+env:
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: ${{ secrets.QUAY_NAMESPACE }}
```

Comment:
I think it's better to keep this visible. It appears in the log anyway so... yep.